internal/brokers/auth/consts.go

```diff
@@ -0,0 +1,25 @@
+// Package auth contains the authentication related code.
+package auth
+
+const (
+	// Granted is the response when the authentication is granted.
+	Granted = "granted"
+	// Denied is the response when the authentication is denied.
+	Denied = "denied"
+	// Cancelled is the response when the authentication is cancelled.
+	Cancelled = "cancelled"
+	// Retry is the response when the authentication needs to be retried (another chance).
+	Retry = "retry"
+	// Next is the response when another MFA (including changing password) authentication is necessary.
+	Next = "next"
```

Comment:
How I wish Go had the same `enum` utilities as Rust... This would look way better, but it's still a very nice change!

Reply:
Cool thanks, I was unsure whether to use prefixes but I think so far the package helps to make it clearer enough :)